build/Linux/pencil

```diff
@@ -1,18 +1,31 @@
 #!/bin/sh
 
-
+# Target for application ini
 APP_INI="/usr/share/evolus-pencil/application.ini"
 
-if [ -x /usr/bin/xulrunner ]; then
+# List of firefox binaries to search
+BINARIES="xulrunner iceweasel firefox firefox-bin"
 
-    /usr/bin/xulrunner "$APP_INI" "$@"
+# Assembling run command
+RUN=""
 
-elif [ -x /usr/bin/iceweasel ]; then
+for BIN in $BINARIES; do
+	BIN_PATH=$(which ${BIN} 2>/dev/null)
 
-    /usr/bin/iceweasel --no-remote --app "$APP_INI" "$@"
+	if [ "$?" == "0" ]; then
+		OPTS="${APP_INI}" "$@"
 
-else
+		if [ "${BIN}" != "xulrunner" ]; then
+			OPTS="--app ${OPTS}"
```

Comment:
Unfortunately, the --no-remote breaks ability to run `firefox` and `pencil` simultaneously because of the file lock on users' profile. If you do that, the classical [firefox error](http://www.lemis.com/grog/Photos/20071117/tiny/broken-firefox.gif) is raised. You would have to create a new profile for pencil to run as a new instance, as seen on [stackoverflow](http://superuser.com/questions/553066/firefox-is-already-running-with-no-remote).

It definitely breaks down pencil for me. Does it work for you?